src/libraries/Common/tests/CoreFx.Private.TestUtilities/System/PlatformDetection.cs

```diff
@@ -63,9 +63,9 @@ public static bool IsDrawingSupported
         public static bool SupportsSsl3 => GetSsl3Support();
 
 #if NETCOREAPP
-        public static bool IsReflectionEmitSupported = RuntimeFeature.IsDynamicCodeSupported;
+        public static bool IsReflectionEmitSupported => RuntimeFeature.IsDynamicCodeSupported;
 #else
-        public static bool IsReflectionEmitSupported = true;
+        public static bool IsReflectionEmitSupported => true;
```

Comment:
Why did these change as part of this PR?

Reply:
Because I need to use `ConditionalFact`, which only supports properties and not fields